data/transit/route/bus.json

```
@@ -2460,11 +2460,12 @@
       }
     },
     {
-      "displayName": "Casablanca Bus",
+      "displayName": "Casabus",
       "id": "casablancabus-1ba109",
       "locationSet": {"include": ["ma"]},
```

```suggestion
      "locationSet": {"include": ["ma"]},
      "matchNames": ["Casablanca Bus"],
```
If this is a name change by business merge or reorganization, usually there still have some element in OSM using old name, and formatter program will gen a empty "Casablanca Bus" entry when run it.
So I suggest you add it as matchName.